docs/about/faq.rst

```diff
@@ -56,42 +56,42 @@ The :doc:`curation pages <../curating>` detail the curation and editing process,
 
 My favorite Gene/Variant is not in CIViC?!?
 -------------------------------------------
-The extensive manual curation required to add evidence statements means that there might not yet be a published evidence statement for every variant or gene of interest. This is precisely why we need community-driven efforts to grow this database. For a gene or variant to appear in CIViC, it must have an evidence statement associated with it. To add evidence statements and begin the discussion about your gene/variant of interest, go :doc:`here <../curating>`!
+The extensive manual curation required to add evidence statements means that there might not yet be an evidence statement for every variant or gene of interest. This is precisely why we need community-driven efforts to grow this database. For a gene or variant to appear in CIViC, it must have an evidence statement associated with it. To add evidence statements and begin the discussion about your gene/variant of interest, go :doc:`here <../curating>`!
 
 May I enter unpublished results from n-of-1 observations?
 ---------------------------------------------------------
 At this time, a publication or abstract is considered a minimum requirement for all evidence statements in CIViC. N-of-1 results from early stage clinical trials or patients treated under compassionate use doctrines are allowed, but only if a case report has been published in a peer reviewed journal. We are considering options for centers that wish to use a local instance of CIViC to capture unpublished individual patient observations. We are also considering other source types including clinical trial records, and ClinVar records.
 
-Why must my contributions be approved by a moderator?
------------------------------------------------------
-In an effort to ensure quality (and prevent automated spam), we require that edits be submitted to the review queue before they are shown as accepted. Moderators are used for this review step to protect the CIViC resource and will approve your revisions as soon as possible. An editor, moderator or administrator may comment on your proposed addition or revision. You will be able to see your new content in a pending state while it awaits review. To expedite the review process, we encourage you to submit high quality evidence (evidence level of validated or clinical) as the top priority. Similarly, proposing evidence statements for a new variant or gene may take longer for the community to review.
+Why must my contributions be approved by an editor?
```

Comment:
Do we need to go formal case on this? Editor?

Reply:
Makes sense